flumine/order/order.py

```diff
@@ -93,6 +93,9 @@ def execution_complete(self) -> None:
         self.date_time_execution_complete = datetime.datetime.utcnow()
         self.update_data.clear()
 
+    def expiring(self) -> None:
+        self._update_status(OrderStatus.EXPIRED)
```

Comment:
There is some logic that needs to be updated to use EXPIRED, off the top of my head:

- `self.status`
- `date_time_execution_complete` because of `self.elapsed_seconds_executable`

Reply:
I'm afraid I don't understand this. Please could you explain in more detail?

Reply:
Because this means the order is now complete (as far as betfair are concerned) we need to add this datetime:

https://github.com/liampauling/flumine/blob/master/flumine/order/order.py#L93

And add `EXPIRED` to this logic:

https://github.com/liampauling/flumine/blob/master/flumine/order/order.py#L168

Reply:
Do you mean adding this line?
https://github.com/liampauling/flumine/blob/65a33d785350d9cfdfa8a9184372e41c23235d10/flumine/order/order.py#L93

`EXPIRED` is already in the logic at https://github.com/liampauling/flumine/blob/master/flumine/order/order.py#L168.

Reply:
Yep 👍